jobs/jobs.go

```diff
@@ -70,8 +75,9 @@ func NewJob(cfg *Config) *Job {
 		restartsRemain:    cfg.restartLimit,
 		frequency:         cfg.freqInterval,
 	}
-	job.InitRx()
+	// job.InitRx()
```

Comment:
can just delete this line

Reply:
Yes, I need to remove these. There might be another somewhere.